adapters/postgres/postgres.go

```diff
@@ -160,6 +149,27 @@ func JoinByRequest(r *http.Request) (values []string, err error) {
 	return joinValues, nil
 }
 
+func SelectByRequest(r *http.Request) []string {
```

Comment:
I got it, but a better place to do this is in `SelectByRequest` function, but don't need rewrite all function. I think if you add `FieldPermissions` on top of `SelectByRequest` looks good. Of course, now `SelectByRequest` need return an error.